src/public/modules/forms/helpers/CsvGenerator.ts

```diff
@@ -38,25 +47,25 @@ module.exports = class CsvGenerator {
    * Insert headers into the CSV file after the meta-data
    * @param {Array} headerLabels array of labels for header row
    */
-  setHeader(headerLabels) {
+  setHeader(headerLabels: (number | string)[]): void {
```

Comment:
I think the headers can only be strings, right?

Reply:
thanks, edited